src/font.h

```diff
@@ -39,14 +42,8 @@ typedef struct unk8012A7EC {
     s32 unk4;
     s32 unk8;
     s32 unkC;
-    u8 unk10;
-    u8 unk11;
-    u8 unk12;
-    u8 unk13;
-    u8 unk14;
-    u8 unk15;
-    u8 unk16;
-    u8 unk17;
+    u8 textColour[4];
```

Comment:
Can we match this with named colours? Like textColurRed, and textColourGreen? Will make reading the code easier so we won't need to keep a mental state of 0 = red, 3 = alpha etc.